packs/data/pathfinder-bestiary-3.db/ioton.json

```diff
@@ -516,7 +503,28 @@
                 "description": {
                     "value": "<p>@Localize[PF2E.PersistentDamage.Mental1d4.success]</p>"
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "key": "Note",
+                        "outcome": [
+                            "success"
+                        ],
+                        "selector": "{item|_id}-damage",
+                        "text": "PF2E.PersistentDamage.Mental1d4.success",
+                        "title": "",
+                        "visibility": "gm"
+                    },
+                    {
+                        "key": "Note",
+                        "outcome": [
+                            "criticalSuccess"
+                        ],
+                        "selector": "{item|_id}-damage",
+                        "text": "PF2E.PersistentDamage.Mental1d4.criticalSuccess",
+                        "title": "",
+                        "visibility": "gm"
```

Comment:
```suggestion
                        "visibility": "gm"
                    },
                    {
                        "key": "Note",
                        "outcome": [
                            "criticalSuccess"
                        ],
                        "selector": "{item|_id}-damage",
                        "text": "PF2E.PersistentDamage.Mental1d4.criticalSuccess",
                        "visibility": "gm"
```

Reply:
Hm, not sure I understand that change?

Reply:
Ok, finally back from camping. If I understand correctly you removed the "titel": ""?
Not sure though if this gets added by default using the implemented note rule element ui.